app/javascript/pages/IndividualEvents/index.js

```diff
@@ -393,14 +410,7 @@ const withData = compose(
 
         return mutate({
           variables: { input: individualEventInput },
-          optimisticResponse: {
-            __typename: 'Mutation',
-            createEditIndividualEvent: {
-              __typename: 'User',
-              id: currentUser.id,
-              individualEvents: R.append(newEvent, currentUser.individualEvents),
-            },
-          },
+          optimisticResponse: buildOptimisticResponse(newEvent, currentUser),
```

Comment:
was this extraction just for readability here? I don't see it being reused

Reply:
Yes, for improved readability and consistency https://github.com/zendesk/volunteer_portal/search?q=buildOptimisticResponse&unscoped_q=buildOptimisticResponse